src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClient.java

```diff
@@ -17,6 +18,7 @@
     url = "${pay-hub.api.url}",
     configuration = PayHubClientConfiguration.class
 )
+@Profile({"integration", "!functional"})
```

Comment:
what are you trying to achieve here?

Reply:
To make this component available in integration profile.

Reply:
it will be available in integration without adding this annotation

Reply:
No, tests fail

Reply:
because integration tests activate `functional` profile as well (while they shouldn't) and you've also added`!functional` here